deploy/example_catalog/cr-app-centos7.json

```diff
@@ -7,6 +7,7 @@
 
     "spec" : {
         "systemdRequired": true,
+        "defaultPersistDirs" : ["/home"],
```

Comment:
just for reference?

Reply:
Just to give it more interesting behavior when using persistent storage. (Should we add some other dirs too?)